app/scripts/models/BaseSaveUploader.js

```diff
@@ -49,6 +49,11 @@ define(['backbone', 'SaveFileMessage', 'SetModelPositionMessage', 'ClearWorkspac
 
             if (ws.get('nodes').length)
                 app.trigger('ws-data-loaded');
+
```

Comment:
I see that this fixes, a problem, but I'd rather you expose this as an event on SaveUploader that App can respond to.  For example, "request-workspace-browser-refresh."